src/tokens/transformation/common/leoParser.js

```diff
@@ -30,12 +34,25 @@ module.exports = {
     // export variable references in effects.
     // At the moment, we just whitelist a few color groups.
     const allowedGroups = ['elevation', 'primary', 'secondary']
-    const effectColors = allowedGroups
+    const universalEffectColors = allowedGroups
       .map((g) =>
-        Object.entries(variables.color['---light'][g]).map(([key, value]) => [
-          `${g}.${key}`,
-          new TinyColor(value.value).toHex8String()
-        ])
+        Object.entries(universalVariables.color['---light'][g]).map(
+          ([key, value]) => [
+            `${g}.${key}`,
+            new TinyColor(value.value).toHex8String()
+          ]
+        )
+      )
+      .flat()
+
+    const layerEffectColors = allowedGroups
+      .map((g) =>
+        Object.entries(layerVariables.color?.['---light']?.[g] || {}).map(
```

Comment:
Maybe worth pulling this into a variable so when we change the name it only needs to be done in one place.

Reply:
I think abstracting to a function, as later suggested, solves this problem.